GCTC-NTGC/gc-digital-talent
# [Docs] Creates GitHub issue template for copy

## 👋 Introduction

This PR adds a GitHub issue templates for copy.

## 🕵️ Details

- [Create issue template for copy](https://github.com/GCTC-NTGC/gc-digital-talent/blob/0106d1598849af8a2e40cd6dc1358073d6b8b4bf/.github/ISSUE_TEMPLATE/%F0%9F%94%A4-copy.md)

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Ensure template has proper heading levels
2. Ensure template is easy to use
